cpu/efm32/efm32-info.mk

```diff
@@ -13,15 +13,15 @@ ifeq (,$(EFM32_INFO))
 endif
 
 # Export variables to use in this build.
-export EFM32_FAMILY = $(word 2, $(EFM32_INFO))
-export EFM32_SERIES = $(word 3, $(EFM32_INFO))
-export EFM32_ARCHITECTURE = $(word 4, $(EFM32_INFO))
+CPU_FAM = $(word 2, $(EFM32_INFO))
+CPU_SERIES = $(word 3, $(EFM32_INFO))
+CPU_ARCH = $(word 4, $(EFM32_INFO))
 
-export EFM32_FLASH_START = $(word 5, $(EFM32_INFO))
-export EFM32_FLASH_SIZE = $(word 6, $(EFM32_INFO))
-export EFM32_SRAM_START = $(word 7, $(EFM32_INFO))
-export EFM32_SRAM_SIZE = $(word 8, $(EFM32_INFO))
+EFM32_FLASH_START = $(word 5, $(EFM32_INFO))
```

Comment:
This variable is already set [here](https://github.com/RIOT-OS/RIOT/pull/13174/files#diff-22687e9fbc629a5d3b242f1cb23315b2R3) but with a different value (apparently). Which one is correct ?
(same question for the 3 other variables below).

Reply:
All the variables here should be removed, I missed that when rebasing I think.

Reply:
done.